rdfunit-junit/src/main/java/org/aksw/rdfunit/junit/RdfUnitJunitStatusTestExecutor.java

```diff
@@ -0,0 +1,36 @@
+package org.aksw.rdfunit.junit;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+
+import org.aksw.rdfunit.exceptions.TestCaseExecutionException;
+import org.aksw.rdfunit.io.reader.RDFModelReader;
+import org.aksw.rdfunit.sources.TestSource;
+import org.aksw.rdfunit.sources.TestSourceBuilder;
+import org.aksw.rdfunit.tests.executors.RLOGTestExecutor;
+import org.aksw.rdfunit.tests.query_generation.QueryGenerationSelectFactory;
+import org.aksw.rdfunit.tests.results.TestCaseResult;
+
+final class RdfUnitJunitStatusTestExecutor extends RLOGTestExecutor {
+
+    public RdfUnitJunitStatusTestExecutor() {
+        super(new QueryGenerationSelectFactory());
+    }
+
+    Collection<TestCaseResult> runTest(RdfUnitJunitTestCase rdfUnitJunitTestCase)
+            throws IllegalAccessException, InvocationTargetException {
+        final TestSource modelSource = new TestSourceBuilder()
+                .setPrefixUri("custom", "rdfunit")
```

Comment:
prefix are used as in turtle syntax (e.g. http://prefix.cc/popular.file.txt) and are meant to shorten the IRIs or as secondary (short) identifiers.
I also use them to override the actual schema IRI  when I want to use a different ontology IRI than the default one
e.g. nif https://github.com/AKSW/RDFUnit/blob/master/data/schemaDecl.csv#L12 skips the default dereferencing and uses the latest version from github.
this way when the user checks against e.g. `nif` the the overridden IRI is used transparently

If we use a `SchemaSource` and a `TestSource` directly in the tests definitions, instead of Jena `Model`s, then we can reuse the prefixes defined in the constructors and avoid the hardcoded prefixes